CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+<!-- The 4.x link will redirect to the new site when it's live -->
+* [Node.js Agent version 4.x](https://www.elastic.co/guide/en/apm/agent/nodejs/current/release-notes-4.x.html)
```

Comment:
Would it be reasonable to also have a link to the local source file (./docs/release-notes/index.md)?

Reply:
Sure. Do you mean a link to _both_ the page and the file on GitHub or _replacing_ this link with a link to the file on GitHub?

Reply:
Either is fine with me.

Reply:
I think bias to the local file...

Reply:
👍  Let's wait to confirm the exact file path that we use in #4524.